internal/manifests/manifests_test.go

```diff
@@ -473,6 +473,51 @@ spec:
 				Expect(err.Error()).To(ContainSubstring("Manifest content of file 99-openshift-machineconfig-master-kargs.json for cluster ID " + clusterID.String() + " exceeds the maximum file size of 1MiB"))
 			})
 
+			It("accept muti-doc yaml file", func() {
+				clusterID := registerCluster().ID
+				fileName := "99-test.yaml"
+				content := encodeToBase64(`---
+first: one
+---
+---
+- second: two
+`)
+				mockUpload(1)
+				expectUsageCalls()
+				response := manifestsAPI.V2CreateClusterManifest(ctx, operations.V2CreateClusterManifestParams{
+					ClusterID: *clusterID,
+					CreateManifestParams: &models.CreateManifestParams{
+						Content:  &content,
+						FileName: &fileName,
+					},
+				})
+				Expect(response).Should(BeAssignableToTypeOf(operations.NewV2CreateClusterManifestCreated()))
+				responsePayload := response.(*operations.V2CreateClusterManifestCreated)
+				Expect(responsePayload.Payload).ShouldNot(BeNil())
+				Expect(responsePayload.Payload.FileName).To(Equal(fileName))
+				Expect(responsePayload.Payload.Folder).To(Equal(defaultFolder))
+			})
+
+			It("Upload fails when one of the yaml in muti-doc yaml file is invalid", func() {
+				clusterID := registerCluster().ID
+				fileName := "99-test.yml"
+				content := encodeToBase64(`---
+first: one
+---
+not a valid YAML: {{ content }}
```

Comment:
I think the {{ content }} here just adds confusion the simple phrase "invalid YAML content" would suffice as such content does not even form a key/value pair with is the minimum requirement for any YAML content (as demonstrated by `first: one`

Reply:
see my previous comment,  "invalid YAML content" is a valid yaml string